modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/model/AuthorizedBusinessAPIResponse.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.identity.integration.test.rest.api.server.application.management.v1.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import org.wso2.identity.integration.test.rest.api.server.api.resource.v1.model.ScopeGetModel;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import javax.validation.Valid;
+
+public class AuthorizedBusinessAPIResponse {
+    private String id;
+    private String name;
+    private String description;
+    private String identifier;
+    private String type;
+    private boolean requiresAuthorization;
+    private List<ScopeGetModel> scopes = null;
+    private List<String> properties = null;
+
+    /**
+     * Set the ID of the API.
+     **/
+    public AuthorizedBusinessAPIResponse id(String id) {
+        this.id = id;
+        return this;
+    }
+
+    @ApiModelProperty(example = "8925afb3-c970-43fd-84f4-2ceb8f77659b", value = "")
```

Comment:
These example values are not needed. This looks like a class that's just copied from a swagger generated one. But the intention seems to be to add this class specifically to be used with the test case.

Therefore, remove unnecessary annonations.

Reply:
Fixed with the commit